Framework/CurveFitting/src/Algorithms/NormaliseByPeakArea.cpp

```diff
@@ -305,8 +307,8 @@ void NormaliseByPeakArea::saveToOutput(const API::MatrixWorkspace_sptr &accumWS,
     for (size_t j = 0; j < npts; ++j) {
       double accumYj = accumWS->y(0)[j];
       double accumEj = accumWS->e(0)[j];
-      double rhsYj = yValues[j];
-      double rhsEj = eValues[j];
+      double rhsYj = yValues->rawData()[j];
+      double rhsEj = eValues->rawData()[j];
```

Comment:
Consider moving the call to `rawData` outside the loop. Just like the old `dataY()` methods (and as done for `accumWS->mutableY()` above), it is better to buffer the reference since the copy-on-write mechanism makes write access more expensive.